ui/packages/components/src/Timeline/TimelineNode/TimelineNodeRenderer.tsx

```diff
@@ -92,6 +92,8 @@ export function renderTimelineNode({
       name = 'Running next attempt...';
     } else if (node.status === 'started') {
       name = 'Running next step...';
+    } else if (node.status === 'failed') {
+      name = 'Function errored';
```

Comment:
This covers the case where a step exhausts all retries and so the error is then thrown to the function itself. This extra check will replace the `"..."` node with `"Function errored"`.